drivers/overlay/ov_utils.go

```diff
@@ -69,6 +71,37 @@ func createVxlan(name string, vni uint32) error {
 	return nil
 }
 
+func deleteInterfaceBySubnet(brPrefix string, s *subnet) error {
+	defer osl.InitOSContext()()
+
+	nlh := ns.NlHandle()
+	links, err := nlh.LinkList()
+	if err != nil {
+		return fmt.Errorf("failed to list interfaces while deleting bridge interface by subnet: %v", err)
```

Comment:
or just make this a log. The cleanup is a best effort. then the function do not need to return any error.